apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -210,8 +237,7 @@ def simulate_game(
             N_s_a = torch.zeros_like(not_dupl_q_values).to("cpu")
             present_state = extract_present_state(state)
             states_dict[to_hash(present_state)] = (
-                (state, actions),
-                cloned_idx_to_idx,
+                [s.to("cpu") for s in possible_states],
```

Comment:
please keep the old code modification (the code in red should replace the code in green)